gtk4/src/prelude.rs

```diff
@@ -8,6 +8,8 @@ pub use crate::actionable::ActionableExtManual;
 pub use crate::builder::BuilderExtManual;
 pub use crate::color_chooser::ColorChooserExtManual;
 pub use crate::combo_box::ComboBoxExtManual;
+pub use crate::custom_filter::CustomFilterExtManual;
```

Comment:
mis-comitted this & the change to `gtk4/src/lib.rs`

Reply:
The commit now has the custom_filter, it might go in another MR tough.

Reply:
I don't see that commit?